src/components/Navbar.tsx

```diff
@@ -50,7 +50,7 @@ const Navbar = () => {
 			<Container maxWidth={false}>
 				<Toolbar disableGutters >
 
-					<Box sx={{ flexGrow: 1, display: { xs: "flex", md: "none" }}}>
+					<Box sx={{ flexGrow: 1, display: { xs: "flex", md: "none" } }}>
```

Comment:
**thought:** I love a good working prettier and I find it personally very important. If you folks want to pair one day to demo it let me know 😊  Otherwise I might find some good resources that I can send your way. It helps making it consistent across team, and less spaces/indentation helps with smaller file sizes and better readability